recipes/cmake/3.x.x/conanfile.py

```diff
@@ -54,6 +54,10 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "doc"))
 
+    def package_id(self):
+        # CMake is a executable-only package, so the compiler version does not matter
```

Comment:
It matters since a system might be missing the runtimes. It won't fix the related issue. CCI will build one MSVC package using the 2015 runtimes, but the target system is a 2017 so it won't be able to execute

Reply:
Building with 2015 is fine, it’ll work with the 2015 - 2019 runtimes.

https://docs.microsoft.com/en-us/cpp/porting/binary-compat-2015-2017?view=vs-2019

Removing the compiler ver may not be a best practice, because if CCI is upgraded to build with 2017+ and the system only had the 2015 runtime, that wouldn’t work.

Reply:
That's a very good point, however these recipes can be used to generate a package for 2013 locally, and the same scenario would cause trouble.

Reply:
To what runtime do the official cmake windows builds link?
The only problem I see with removing version in package_id, is the loss of reproduceability as the ci can build cmake using whatever version of msvc it pleases.

Reply:
Would it make sense to only allow static runtime? Or should we just build all variants of cmake and don't bother with limiting anything?

Reply:
https://gitlab.kitware.com/cmake/cmake/-/tree/master/.gitlab/ci looks like vs2019

Reply:
Looks like the official build uses the static runtime.  This is the [latest release](https://github.com/Kitware/CMake/releases/download/v3.17.3/cmake-3.17.3-win64-x64.zip):

![image](https://user-images.githubusercontent.com/15334377/83337985-d5d7ce80-a28d-11ea-8a6a-5ba33464c7aa.png)

Reply:
I'd prefer to build static runtime for CMake (and built tools in general), this will avoid a dependency on Visual Studio CRT redist, which is not necessarily available (especially, if I am using alternate compiler).